service/eventbridge/api_client.go

```diff
@@ -512,3 +533,32 @@ func addRequestResponseLogging(stack *middleware.Stack, o Options) error {
 		LogResponseWithBody: o.ClientLogMode.IsResponseWithBody(),
 	}, middleware.After)
 }
+
+type EndpointDisableHTTPSMiddleware struct {
```

Comment:
**correctness**: Same question, why public?

Reply:
will privatize in next PR